Loop/Managers/Alerts/AlertStore.swift

```diff
@@ -22,13 +22,16 @@ public class AlertStore {
         
     private let log = DiagnosticLog(category: "AlertStore")
     
-    public init(storageFileURL: URL? = nil) {
+    public init(storageDirectoryURL: URL? = nil) {
         managedObjectContext = NSManagedObjectContext(concurrencyType: .privateQueueConcurrencyType)
         managedObjectContext.mergePolicy = NSMergeByPropertyStoreTrumpMergePolicy
         managedObjectContext.automaticallyMergesChangesFromParent = true
 
         let storeDescription = NSPersistentStoreDescription()
-        if let storageFileURL = storageFileURL {
+        if let storageDirectoryURL = storageDirectoryURL {
+            let storageFileURL = storageDirectoryURL
+                .appendingPathComponent("AlertStore")
```

Comment:
[nit] Note of the other store directories or files use the "Store" prefix. Consider "Alerts/Alerts.sqlite" or just "Alerts.sqlite" (unless there is a need for the directory).

Reply:
Good point, but these names already are out there (just moved to this file, because I felt like the responsibility for this name should live here, not the caller).  Since this is a [nit] I'm going to leave it as-is.